### PR TITLE
Fix english in stdenv.xml

### PR DESCRIPTION
**Pull request by guibou**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: doc/stdenv.xml [hunk 1]
```diff
@@ -2500,7 +2500,7 @@ addEnvHooks "$hostOffset" myBashFunction
      <listitem>
       <para>
        This hook will make a build pause instead of stopping when a failure
-       happen. It prevents nix to cleanup the build environment immediatly and
+       happen. It prevents nix from cleaning up the build environment immediatly and
```
Comment (veprbl) [inline doc/stdenv.xml]:
```suggestion
       happen. It prevents nix from cleaning up the build environment immediately and
```

**Comment by worldofpeace** [comment]
I actually spotted a few more after reading this https://github.com/worldofpeace/nixpkgs/commit/15280c718682e14982c3590194b206345d483161

**Comment by Mic92** [comment]
backport: fbfe10f0a17

**Comment by Mic92** [comment]
@worldofpeace I have not backported yours. There were some merge conflicts.

**Comment by worldofpeace** [comment]
I thought as much, thanks anyway.

**Pull request merged**